### DISCUSSION TITLE
Can someone help me pls?

### DISCUSSION THREAD

**Discussion by HungTran1412** [community/community#125162] [Repositories] [answered]
### Select Topic Area

Question

### Body

my main account was banned. I don't know why when I upload my project to github. I just finished syntax to upload my project, I turned back the github homepage when I saw my account was locked

**Comment by ankit-orion** [comment]
That sounds really frustrating. Sometimes accounts can get flagged by mistake. Your best bet is to contact GitHub support directly to find out why your account was banned and to get it resolved. Make sure to provide them with any relevant details about your project and account activity.

Hope it gets sorted out quickly

**Comment by queenofcorgis** [comment, marked as answer]
Hi @HungTran1412 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). 

It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
